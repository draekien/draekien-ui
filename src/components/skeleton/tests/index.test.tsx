import { render } from '@testing-library/react';
import React from 'react';
import Skeleton from '..';

describe('Skeleton component', () => {
  test('should match snapshot', () => {
    const { container } = render(<Skeleton />);

    expect(container).toMatchSnapshot();
  });

  test('should match snapshot with custom props', () => {
    const { container } = render(
      <Skeleton
        height="1rem"
        width="1rem"
        color="primary"
        backgroundColor="p-000"
        boxShadow="lg"
        borderRadius="lg"
        fullWidth
      />
    );

    expect(container).toMatchSnapshot();
  });
});
