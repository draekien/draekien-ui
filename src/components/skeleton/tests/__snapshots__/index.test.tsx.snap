// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton component should match snapshot 1`] = `
<div>
  <div
    class="css-1bkbd36-Skeleton"
  />
</div>
`;

exports[`Skeleton component should match snapshot with custom props 1`] = `
<div>
  <div
    class="css-1gjv3gb-Skeleton"
  />
</div>
`;
